apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -71,6 +73,7 @@ export const ActionsBanner = ({disabled = false}: {disabled: boolean}) => {
 
   const handleOnPressReceive = () => {
     if (isSingle) {
+      selectedAddressChanged(nextReceiveAddress)
```

Comment:
missing dispatch when single causing blank single address - I forgot picking this commit in the 1st PR.